masui/build.gradle

```diff
@@ -59,7 +55,7 @@ bintray {
         repo = 'maven'
         name = 'masui'
         userOrg = 'caapim'
-        desc = 'Provides resources to implement a user login dialog, Social Login, One-Time Password, Proximity Login (QR code and BLE), Enterprise Browser and Fingerprint'
+        desc = 'A core UI framework with resources to implement a user login dialog. For example, Social Login, One-Time Password, Proximity Login (QR code and BLE), Enterprise Browser, and Fingerprint.'
```

Comment:
The way this is currently phrased makes it sound like it's only to implement a user login screen with the rest being places where you can use it. How about "A core UI framework with resources to help implement features such as User Login, Social Login, One-Time Password, Proximity Login (QR code and BLE), Enterprise Browser, and Fingerprint Unlock."?